scss/_patterns_lists.scss

```diff
@@ -175,7 +175,8 @@ $list-step-bullet-margin: $sph--x-large;
     }
 
     &.is-split .p-list__item:last-of-type {
-      border-bottom: 1px solid $color-mid-light;
+      // this can be updated to use $border after merging #5101
```

Comment:
Nit: Items that require future work should probably be prefixed with "TODO" (and optionally, a link to an Issue)

Reply:
Fair point! I've just updated this to use `$border` and remove that comment since #5112 (not 5101... that was a typo) was merged this morning; but I'll keep this in mind for future :)